lib/utilities.js

```diff
@@ -42,20 +43,33 @@ function isDockerInstalled(outputChannel, cb) {
         localExecCmd('cmd.exe', ['/c', 'docker', '-v'], outputChannel, function (err) {
             if (err) {
                 vscode.window.showErrorMessage('Docker isn\'t installed, please install Docker firstly!');
-                cb();
-            } else {
                 cb(err);
+            } else {
+                cb()
             }
         });
     }
 }
 
+function isAnsibleInstalled(outputChannel, cb) {
+    child_process.exec("type ansible").on('exit', function (code) {
```

Comment:
why not use localExecCmd here?

Reply:
According to my testing, there would be run-time exception if the command doesn't exist.